biggraphite/drivers/lucene.py

```diff
@@ -88,12 +86,13 @@ def translate_to_lucene_filter(components):
         parent = ""
         for component in components[:-1]:
             parent += component[0] + "."
-        must_list.append(FIELD_MATCH_VALUE % ('parent', parent))
+        must_list.append(json.dumps(
+            {"field": "parent", "type": "match", "value": parent}))
     else:
         must_list = _build_filters(components)
         # Restrict length by matching the END_MARK
-        must_list.append(FIELD_MATCH_VALUE %
-                         (_component_name(len(components)), END_MARK))
+        must_list.append(json.dumps(
+            {"field": _component_name(len(components)), "type": "match", "value": END_MARK}))
```

Comment:
could we do the json.dumps() were we join the constraints instead ? (`LUCENE_FILTER % ",\n            ".join(must_list)`)